R/pkg/R/functions.R

```diff
@@ -3404,19 +3404,27 @@ setMethod("collect_set",
 #' Equivalent to \code{split} SQL function.
 #'
 #' @rdname column_string_functions
+#' @param limit determines the length of the returned array.
```

Comment:
shall we mention this is an optional param?

Reply:
going to include this in the `@details` section, as other functions like `ltrim` handle optionality of one of its params there.